pennylane/ops/qubit/special_unitary.py

```diff
@@ -279,6 +279,11 @@ class SpecialUnitary(Operation):
     differentiation techniques if the JAX, Torch or TensorFlow interface is used.
     See the theoretical background section below for details.
 
+    .. seealso:: 
```

Comment:
It might be good for this to go above the **Details:** as it's a bit hidden down here.